### PR TITLE
Refactor entity init arrays

### PR DESCRIPTION
**Pull request by Xeeynamo**
Standardize names, plus give proper names to almost all the imported EInit arrays

### CODE REVIEW

File: config/symbols.hd.stcen.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
-g_InitializeData0 = 0x80180410;
-g_InitializeEntityData0 = 0x8018041C;
-g_eMariaInit = 0x80180428;
+g_EInitObtaineable = 0x80180410;
```
Comment (sozud) [inline config/symbols.hd.stcen.txt:1]: obtainable

File: config/symbols.pspeu.stwrp.txt [hunk 2]
```diff
@@ -98,14 +98,14 @@ D_801801B8 = 0x09246D88;
...
-g_InitializeEntityData0 = 0x09246EB0;
-g_EInitGeneric = 0x09246EC0;
-g_InitDataEnt13 = 0x09246ED0;
-g_EntityUnkId12Init = 0x09246EE0;
-g_eInitGeneric2 = 0x09246EF0;
-g_eDamageDisplayInit = 0x09246F00;
+g_EInitBreakable = 0x09246E90;
+g_EInitObtaineable = 0x09246EA0;
+g_EInitParticle = 0x09246EB0;
+g_EInitInteracteable = 0x09246EC0;
```
Comment (sozud) [inline config/symbols.pspeu.stwrp.txt:104]: interactable

File: src/st/nz0/e_init.c [hunk 3]
```diff
@@ -159,39 +159,39 @@ PfnEntityUpdate OVL_EXPORT(EntityUpdates)[] = {
...
+u16 g_EInitBloodyZombie[] = {0x8005, 0x0001, 0x004A, 0x0209, 0x000D};
+u16 g_EInitSkeleton[] = {0x8006, 0x0001, 0x004E, 0x020C, 0x004B};
+u16 g_EInitSkeletonPieces[] = {0x8006, 0x0000, 0x004E, 0x020C, 0x0002};
+u16 g_EInitSkeletonBone[] = {0x8006, 0x0015, 0x004E, 0x020C, 0x004C};
+u16 g_EInitSpittleBone[] = {0x8007, 0x0034, 0x0050, 0x020D, 0x0051};
+u16 g_EInitSpittleBoneSpit[] = {0x8007, 0x0040, 0x0050, 0x020D, 0x0052};
+u16 g_EInitTableWithGlobe[] = {0x8008, 0x0000, 0x004B, 0x0219, 0x0005};
+u16 g_EInitLifeMaxTank[] = {0x8009, 0x0000, 0x004F, 0x0211, 0x0005};
+u16 g_EInitPrizeContainer[] = {0x800A, 0x0000, 0x0053, 0x0215, 0x0005};
+u16 g_EInitSubwpnCloche[] = {0x800C, 0x0000, 0x004C, 0x021D, 0x0005};
```
Comment (sozud) [inline src/st/nz0/e_init.c:186]: I'd never heard of the term cloche but it seems appropriate. Not sure if there's a more common term in English or not. Image from google image search:
image

**Review by sozud — APPROVED** [review]

File: src/st/nz0/e_init.c (hunk 3 above)
Comment (JoshSchreuder) [reply src/st/nz0/e_init.c:186]: I only know it as those things from fancy restaurants to hide the food
image

**Pull request merged**